index.js

```diff
@@ -41,31 +52,46 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 4", "color: red") 
 
-// console.log(sameSameButDifferent(5, 5)) 
-// // => "same same"
-//
-// console.log(sameSameButDifferent(123, "123")) 
-// // => "same same (but different)"
-//
-// console.log(sameSameButDifferent(5, 7)) 
-// // => "different"
-//
-// console.log(sameSameButDifferent(123, "122")) 
-// // => "different"
-// console.log("%c----------", "color: red") 
+function sameSameButDifferent(num, maybenum) {
+  if (num === maybenum) {
+    return "same same"
+  } else {
+    return "same same (but different)"
+  }
```

Comment:
You're on the right track here, and you've got the right idea with the `return` keyword - just missing one other condition:

```js
  if (num === maybenum) {
    return "same same"
  } else if(num === parseInt(maybeNum)) {
    return "same same (but different)"
  } else {
    return different
  }
```